servers/zts/src/main/java/com/yahoo/athenz/zts/ZTSImpl.java

```diff
@@ -3643,8 +3643,8 @@ Principal createPrincipalForName(String principalName) {
     @Override
     public ResourceAccess getResourceAccessExt(ResourceContext ctx, String action, String resource,
             String trustDomain, String checkPrincipal) {
-        
-        final String caller = "getaccessext";
+
+        final String caller = ctx.getApiName();
```

Comment:
Please note that I have changed the caller from "getaccessext" to "getresourceaccessext". Unlike the call in putServiceIdentitySystemMeta, there is no call to verifyAuthorizedServiceOperation so this should affect anything.